services/base-service/models/expert/api.js

```diff
@@ -36,27 +63,37 @@ async function sanitize(req, res) {
         delete doc["@graph"][i]["totalAwardAmount"];
       }
     }
-    res.status(200).json(doc);
+    return doc;
   }
 }
 
 // this path is used instead of the defined version in the defaultEsApiGenerator
 router.get(
-  '/*',
+  '/expert/*',
   async (req, res, next) => {
+
+    const acceptHeader = req.headers.accept;
+
     logger.info(`GET ${req.url}`);
-    let id = '/'+model.id+decodeURIComponent(req.path);
+    let id = '/' + model.id + decodeURIComponent(req.path);
     try {
       let opts = {
-        admin : req.query.admin ? true : false,
+        admin: req.query.admin ? true : false,
       }
       res.thisDoc = await model.get(id, opts);
-      next();
-    } catch(e) {
+      // next();
+    } catch (e) {
       res.status(404).json(`${req.path} resource not found`);
     }
+    let doc = await sanitize(req, res);
+    doc = res.thisDoc;
+    // Conditionally reformat based on the Accept header
+    if (acceptHeader && acceptHeader.includes('site-farm')) {
```

Comment:
What are we telling their team to put in as the Accept? `application/ld+json profile="#site-farm"` ?

Reply:
We should be able to find a library to parse the accept header format, make this a little more precise.

Reply:
So the middleware checks for conditions and bails or not.